### DISCUSSION TITLE
I am having a problem with date validation.

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2564] [Help] [answered]
Hi, I am having a problem when using the data validation rules. 
I would like a date field to have two restrains. 
1) cant be a date before the request was submitted
2) cant be a date before the draftsman started work on it. 

when I put in these two restrains it now makes the value required in the field when setting the date the draftsman started working on it.
I want to leave the finished date blank for now.  am I doing something wrong here?

Is there a way to add if the input is null then don't apply the validation?

![image](https://user-images.githubusercontent.com/80058238/132384113-87f6870b-defe-4f84-b3a0-c4a659d69e69.png)
![image](https://user-images.githubusercontent.com/80058238/132384121-9b9352ec-d0d3-45cb-80e5-48b2ae85fbbb.png)

**Comment by mjashanks** [comment, marked as answer]
@KernOil - apologies for the delay - missed this one! 

You can do conditional validations, using "Configure Conditions"

![image](https://user-images.githubusercontent.com/3524181/132838423-fd4fcc64-66e5-4d26-9f81-ed7cbc058bfe.png)

**Reply by KernOil** [reply]
Hi @mjashanks This is a good option to look at thank you for the suggestion.

**Reply by KernOil** [reply]
Works perfect thank you!

**Comment by sergey-dme** [comment]
Hey @KernOil are your validation conditions for Date/Picker still working in the updated app?
Mine are not working, if there are more than one validation condition exist for Date/Time picker.
https://github.com/Budibase/budibase/issues/3675

**Reply by KernOil** [reply]
@sergey-dme Hi, I have moved away from BB for my application. could never get the dates and time to work consistently with my SQL db.
